main.go

```diff
@@ -38,27 +48,34 @@ func main() {
 		}
 	})
 
-	// Обработчик для API запросов
-	http.HandleFunc("/api/tasks", GetTasksHandler)     // Получение списка задач
-	http.HandleFunc("/api/task/done", TaskDoneHandler) // Завершение задачи
-	http.HandleFunc("/api/nextdate", NextDateHandler)  // Получение следующей даты
-	http.HandleFunc("/api/task", func(w http.ResponseWriter, r *http.Request) {
-		switch r.Method {
-		case http.MethodGet:
-			GetTaskByIDHandler(w, r) // Получение задачи по ID
-		case http.MethodPost:
-			AddTaskHandler(w, r) // Добавление новой задачи
-		case http.MethodPut:
-			UpdateTaskHandler(w, r) // Обновление задачи
-		case http.MethodDelete:
-			DeleteTaskHandler(w, r) // Удаление задачи
-		default:
-			http.Error(w, "Метод не поддерживается", http.StatusMethodNotAllowed)
-		}
+	// /api/tasks - Получение списка задач (только GET запросы)
+	http.HandleFunc("GET /api/tasks", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
чтобы тут выглядело лучше делают так
```suggestion
	http.HandleFunc("GET /api/tasks", GetTasksHandler(db))
```

А хендлер
```golang
func GetTasksHandler(db *database.Database) func(w http.ResponseWriter, r *http.Request) {
	return func(w http.ResponseWriter, r *http.Request) {
		...
	}
}